src/processTargets/index.ts

```diff
@@ -142,6 +155,45 @@ function processRangeTarget(
   );
 }
 
+function processColumnTarget(
+  target: RangeTarget,
+  anchorTarget: TypedSelection,
+  activeTarget: TypedSelection,
+  isForward: boolean
+) {
+  const anchorLine = targetToLineLimitPosition(
+    anchorTarget,
+    isForward,
+    !target.excludeAnchor
+  );
+  const activeLine = targetToLineLimitPosition(
+    activeTarget,
+    !isForward,
+    !target.excludeActive
+  );
+  const startLine = isForward ? anchorLine : activeLine;
+  const endLine = isForward ? activeLine : anchorLine;
+  const anchorSelection = anchorTarget.selection.selection;
+  const startCharacter = anchorSelection.start.character;
+  const endCharacter = anchorSelection.end.character;
+  const results: TypedSelection[] = [];
+
+  for (let i = startLine; i <= endLine; ++i) {
+    results.push({
+      selection: {
+        selection: new Selection(i, startCharacter, i, endCharacter),
```

Comment:
I think most consistent would be to use the anchor and active of the anchor selection rather than the start and end, so that the generated selections have the same direction has the anchor

Reply:
done.